### DISCUSSION TITLE
Point Cloud with Crazy flies

### DISCUSSION THREAD

**Discussion by Paramjit26** [bitcraze/discussions#787] [General]
I am working on summer Project, I want to know how to create point cloud using Crazyflie. Can anybody provide guidance

**Comment by krichardsson** [comment]
Could you please specify a bit more what you want to do, what equipment you have and so on? Your description is very vague and could mean a lot of different projects.

**Comment by knmcguire** [comment]
Have you seen this example by any chance? https://github.com/bitcraze/crazyflie-lib-python/blob/master/examples/multiranger/multiranger_pointcloud.py

**Reply by Paramjit26** [reply]
Thank You Ma'am
